staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go

```diff
@@ -174,7 +174,7 @@ func (c *Client) Send(ctx context.Context, request *http.Request) (*http.Respons
 			regionalErrStr = regionalError.Error().Error()
 		}
 
-		klog.V(5).Infof("Send.sendRegionalRequest failed to get response from regional host, error: '%s'. Ignoring the result.", regionalErrStr)
+		klog.V(5).InfoS("Send.sendRegionalRequest failed to get response from regional host. Ignoring the result.", "error", regionalErrStr)
```

Comment:
error->err, suggested in https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```
With the exception of acronyms like "IP" and the standard "err", don't shorten names. Don't use addr, do use address.
```